package.json

```diff
@@ -1,48 +1,53 @@
 {
-  "description": "A Polymer-based web component for images requiring authentication",
+  "name": "@d2l/image",
+  "version": "3.3.1",
+  "description": "A Lit web component to render images requiring authentication",
   "keywords": [
     "d2l",
     "image"
   ],
+  "main": "d2l-image.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Brightspace/d2l-image.git"
   },
   "homepage": "https://github.com/Brightspace/d2l-image#readme",
-  "name": "d2l-image",
   "scripts": {
-    "lint": "npm run lint:wc && npm run lint:js",
-    "lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html",
-    "lint:wc": "polymer lint",
-    "test:local": "polymer test --skip-plugin sauce",
-    "test": "polymer test --skip-plugin local"
+    "lint": "npm run lint:eslint && npm run lint:lit",
+    "lint:eslint": "eslint . --ext .js,.html",
+    "lint:lit": "lit-analyzer \"{components,test}/**/*.js\" --strict --rules.no-unknown-tag-name off",
+    "start": "web-dev-server --open demo/ --node-resolve --watch",
+    "test": "npm run lint && npm run test:headless",
+    "test:headless": "web-test-runner --files ./test/**/*.test.js --node-resolve --group default"
```

Comment:
I don't think `--group default` is needed since there aren't any groups.

Reply:
Ah, good catch! I copied a lot of these commands over from another repo.